### DISCUSSION TITLE
yaml path between the tabs ant the text

### DISCUSSION THREAD

**Discussion by EdgarLip** [redhat-developer/vscode-yaml#926] [Q&A] [answered]
HI, 
Trying to understand if there is a way to change the behavior of yaml file preview in VsCode.
as of now when yaml file is opened , between the tab of the file and the file's text there is a small line that shows the path of the file in the 
file system and the path where i currently at in the yaml file ( yaml path).
this path becomes too long and it is getting hard to read it, can it be changed to see only the path of the yaml ? 

p.s
the Yaml extension that i am using : redhat.vscode-yaml

![q-yaml-path](https://user-images.githubusercontent.com/52099499/235443202-63fa7449-cc75-472f-aedc-1d338352105a.png)

#yaml  #yamlpath #yaml hierarchy

**Comment by gorkem** [comment]
This is not behaviour specific to YAML. You can customize this behaviour as explained on the [breadcrumbs](https://code.visualstudio.com/docs/editor/editingevolved#_breadcrumbs) section on vscode's documentation.

**Reply by EdgarLip** [reply, marked as answer]
yes - your are correct . 
the path that i found it at is : file (top bar)-> preferences -> settings ->  then a setting tab will pop up - > in the search bar  type 
"breadcrumbs " - > Breadcrumbs: File Path -> set to off. 

adding result pic 
![Capture1111](https://github.com/redhat-developer/vscode-yaml/assets/52099499/da6bb7f4-3d51-4f60-8d38-0e4ee53c422f)
